### PR TITLE
[WIP] started lesson intro

### PR DESCRIPTION
**Pull request by ztephm**
Started the intro @ohasselblad . Posting for work in progress tag, please don't merge b/c there's more to add

### CODE REVIEW

**Comment by ohasselblad** [comment]
ref #160

**Comment by ztephm** [comment]
Hi @ohasselblad when you're reviewing the Quantification section might need special attention!

**Comment by ztephm** [comment]
hi @andrewxhill @auremoser @namessanti when u have time want to let me know if u have edits? Esp in Quantification section!

File: courses/06-intermediate-design/lesson-1.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+layout: lesson
+title:  "Lesson 1"
+subtitle: "Which kind of map should I make?"
+course: "Intermediate Map Design"
+course_slug: "06-intermediate-design"
+continue_link:
+tweet_text: "Step by step is the way to go. I've finished the second map academy design course. Check it out!"
+vizjson: "http://documentation.cartodb.com/api/v2/viz/e667d364-d5ff-11e3-a78a-0edbca4b5057/viz.json"
+---
+
```
Comment (ohasselblad) [inline courses/06-intermediate-design/lesson-1.md]: I'd put a map or other visual here

File: courses/06-intermediate-design/lesson-1.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+There is no one accepted answer for this, but users should be able to see a clear hierarchy between your map's visual elements. This topic was discussed in the [third lesson](http://academy.cartodb.com/courses/02-design-for-beginners/lesson-3.html) of the beginner design course.
+
+**Do you want to map changes in your data over time?**
+
+Animated maps are possible with CartoDB's [Torque](http://docs.cartodb.com/cartodb-editor.html#map-wizards).
+
+## Simple Wizard
+
+When you connect a dataset to your CartoDB account, you'll see it's Map View is automatically styled according to our simple wizard.
+
```
Comment (ohasselblad) [inline courses/06-intermediate-design/lesson-1.md]: Do you need all of those screenshots? I think that users understand that it works across all geometries? Also, what about including captions below the images?

File: courses/06-intermediate-design/lesson-1.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+![line_layer]({{site.baseurl}}/img/course6/lesson1/line_layer.png)
+
+![point_layer]({{site.baseurl}}/img/course6/lesson1/point_layer.png)
+
+![polygon_layer]({{site.baseurl}}/img/course6/lesson1/polygon_layer.png)
+
+You can change color and other properties of your markers, lines, or polygons, but notice that the changes apply universally. In this earthquake map, how can you tell which ones were strongest? You can't since there are no differences in marker size or color to indicate magnitude. The Simple wizard option works well for maps that show location only, and you can extend it by applying composite operations on the markers. But if you also want to show other attributes from the dataset, other wizards are a better choice. If you find that your location points are too close together to be readable, consider using a cluster or density map instead.
+
+![earthquake_map]({{site.baseurl}}/img/course6/lesson1/earthquakes.png)
```
Comment (ohasselblad) [inline courses/06-intermediate-design/lesson-1.md]: be sure to include OSM attribution on the screen captures

**Comment by ztephm** [comment]
hi @ohasselblad only non-osm attributed map lesson imgs are our midnight commander ones fyi.

**Comment by iriberri** [comment]
This looks awesome. :revolving_hearts:

**Comment by ztephm** [comment]
hi @ohasselblad no captions for now https://github.com/CartoDB/academy/issues/193

ref: blog post about this lesson https://github.com/CartoDB/blog/pull/351

File: courses/06-intermediate-design/lesson-1.md [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+
+Did the Northeast have more storms than the Midwest? How many storms occurred around Washington, DC? Because of the large number of overlapping points, it's nearly impossible to tell. Our audience can't easily pick out general or specific information from this map.
+
+The Cluster wizard solves this by reducing the number of points. It lays an invisible grid over our map, then creates one marker to symbolize all of the points that fall into each grid cell. Larger markers represent more points, and a marker's number shows how many points it has aggregated. 
+
+You can choose the size of the grid cell by choosing how many buckets your data is divided into in the wizard's pulldown menu. A smaller bucket number means a larger aggregation area, so more points will potentially be included in each grid cell. The grid's pixel size stays the same on zoom. 
+
+Zooming in means a smaller area of your map is included in each grid cell; zooming out means a larger area is included. Notice how points fall into different aggregation grid cells depending on zoom level below.
+
+![cluster_zoom]({{site.baseurl}}/img/course6/lesson1/cluster_zoom_2.gif)
```
Comment (ohasselblad) [inline courses/06-intermediate-design/lesson-1.md]: @ztephm could you give this one some more design? try a color other than the orange, reduce the border width, etc.

File: courses/06-intermediate-design/lesson-1.md [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+subtitle: "Which kind of map should I make?"
+course: "Intermediate Map Design"
+course_slug: "06-intermediate-design"
+continue_link:
+tweet_text: "Step by step is the way to go. I've finished the second map academy design course. Check it out!"
+vizjson: "https://documentation.cartodb.com/api/v2/viz/ea5e3532-4753-11e5-90e9-0e9d821ea90d/viz.json"
+---
+
+## Which kind of map should I make?
+
```
Comment (ohasselblad) [inline courses/06-intermediate-design/lesson-1.md]: How about a map towards the top that's really well designed? maybe a map from the community?

**Comment by ohasselblad** [comment]
ok @ztephm this is almost ready to be merged! just a few small things to fix then let's do it today :+1:

File: courses/06-intermediate-design.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+time: 45 minutes
+description_short: "Learn more about cartography and the pros and cons of thematic map types." 
+description_long: "In this lesson you'll learn how to choose the type of thematic map that communicates your dataset's information best. You'll also learn how to use color and typography to make it beautiful! When you have finished this course, you will have built upon the skills you gained in Introduction to Map Design."
+prerequisite:
+    - Introduction to Map Design
+    - A modern browser like Chrome, Firefox, or Safari
+    - Reliable Internet access
+    - A desire to learn!
+published: true
+vizjson: "https://documentation.cartodb.com/api/v2_1/viz/9333ec12-84bb-11e4-b367-0e9d821ea90d/viz.json"
```
Comment (ohasselblad) [inline courses/06-intermediate-design.md]: And this viz.json doesn't appear to be working

**Comment by ztephm** [comment]
@ohasselblad re cluster gif I purposely left the bolder strokes on larger markers in this revision.

Reason is there is still some visual clutter when larger markers are surrounded by a lot of un-clustered small markers, so bolder outline helps cut down on that. See if u like the negative-space effect from making strokes same color as basemap, I think it emphasizes the larger markers only slightly from the smaller ones, so doesn't make the stroke too prominent.

**Comment by ztephm** [comment]
Hi, we got permission from gkudos to use their map @ohasselblad

**Comment by ohasselblad** [comment]
let's ship it :)

**Comment by andrewxhill** [comment]
woot!

On Thu, Aug 27, 2015 at 5:36 PM, ztephm <notifications@github.com> wrote:

> Merged #174 <https://github.com/CartoDB/academy/pull/174>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CartoDB/academy/pull/174#event-394497154>.
>



-- 

andrewxhill <http://twitter.com/andrewxhill>
http://cartodb.com

**Pull request merged**